docs-source/spring/content/platform/config/_index.md

```diff
@@ -1,32 +1,395 @@
 ---
-title: "Configuration"
+title: "Spring Config Server"
 ---
 
-Oracle Backend for Spring Boot and Microservices includes Spring Cloud Config which provides server- and client-side support for externalized
-configurations in a distributed system. The Spring Cloud Config server provides a central place to manage external properties for applications
-across all environments.
+Oracle Backend for Spring Boot and Microservices includes Spring Cloud Config which provides server- and client-side support for externalized configurations in a distributed system. The Spring Cloud Config server provides a central place to manage external properties for applications across all environments.
 
 The Spring Cloud Config server is pre-configured to work with the Spring Boot Eureka service registry, and it is configured to store the
```

Comment:
Change to:

... configured to store the following:

Reply:
Fixed

Reply:
I think I fixed it